docs/sources/clients/promtail/stages/limit.md

```diff
@@ -0,0 +1,55 @@
+---
+title: limit
+---
+# `limit` stage
+
+The `limit` stage is a rate limiting stage throttles logs based on several options. 
+
+## Limit stage schema
+
+```yaml
+limit:
+  # The allowed ingestion rate limit in lines per second
+  [rate: <int>]
+
+  # The quantity of allowed ingestion burst lines
+  [burst: <int>]
```

Comment:
```suggestion

This pipeline stage places limits on the rate or burst quantity of log lines that Promtail pushes to Loki.
The rate and burst limits are equivalent to the limits specified by the distributor configuration parameters  `ingestion_rate_mb` and `ingestion_burst_size_mb`, as defined in [limits_config](../../../../configuration/#limits_config).

```yaml
limit:
  # The rate limit in lines per second that Promtail will push to Loki
  [rate: <int>]

  # The cap in the quantity of burst lines that Promtail will push to Loki
  [burst: <int>]
```

Reply:
I made an edit to the comment above so it now reads
```
This pipeline stage places limits on the rate or burst quantity of log lines that Promtail pushes to Loki.
The concept of having distinct burst and rate limits mirrors the approach to limits that can be set for Loki's distributor component:  `ingestion_rate_mb` and `ingestion_burst_size_mb`, as defined in [limits_config](../../../../configuration/#limits_config).

```